todo/src/InputWithLabel.js

```diff
@@ -0,0 +1,15 @@
+import React, {useEffect, useRef} from "react";
+
+const InputWithLabel = (props) => {
+    const inputRef = useRef();
+    useEffect(() =>
+        {inputRef.current.focus()}, []);
```

Comment:
Figure out how to use the isFocused prop in this method. The road to react book has a good example of it. You can do this!! 👍 Holler at me if anything